dali/test/python/test_external_source_dali.py

```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Technicaly, you do edit this file, so this should be `2020-2021`, right? ;)

Reply:
Technically yes, but I just took the scope of the edits before adding this.